.eslintrc

```diff
@@ -39,7 +39,7 @@
     "space-before-function-paren": [2, {
       "anonymous": "always", "named": "never"
     }],
-    "spaced-line-comment": [2, "always"],
+    "spaced-comment": [2, "always"],
```

Comment:
That sounds good, I think it makes sense!
I would just want to keep parity at the very minimum with the rules that we currently employ.